support/mock/mockrt.go

```diff
@@ -185,7 +187,22 @@ func (rt *Runtime) Send(toAddr addr.Address, methodNum abi.MethodNum, params abi
 	if rt.inTransaction {
 		rt.Abort(exitcode.SysErrorIllegalActor, "side-effect within transaction")
 	}
-	panic("implement me")
+	sentMsg := &message{
+		to:     toAddr,
+		method: methodNum,
+		params: params,
+		value:  value,
+	}
+	if rt.sendingMessage == nil {
+		rt.t.Fatalf("unexpected message send: %v", sentMsg.Key())
```

Comment:
Using a half-constructed expectedMessage here is a bit yuk. Just format the parameters into this string directly. Also no need for `fatal`.

Reply:
If I add a `String()` method to message can we use that? It will save a considerable amount of typing (e.g. the line below this will need to be triple line wrapped)